src/components/Comments/Comments.jsx

```diff
@@ -52,6 +52,19 @@ const Comments = () => {
         
     };
 
+    const handleDelete = (commentId, user) => {
+
+        if(selectedUser === user){
+        newsApi.delete(`comments/${commentId}`)
+        .then(() => {
+            confirm('Comment deleted');
+        })
+        .catch((error) => {console.log(error)})
+    } else {
+        alert('You cannot delete another users comment')
+    }
+    }
+
     if(isLoading){
         return <Loading />;
     }
```

Comment:
Entirely optional: You may want to look into something called useMemo for this users array you've created - or you can put it into a state and set the state in a useEffect - doing this here means this code will run on every re-render